Processes/DevRel-Publication-Process.md

```diff
@@ -1,30 +1,34 @@
-# Process: Publish New or Updated Content to the Dev Portal
+# Process: Write, Edit, Update Content using GitHub
 
 ## Overview
-This document provides instructions on how to publish new or updated content to the EOS Dev Portal (docs.eosnetwork.com).
+This document provides instructions on how to publish new or updated content to EOS Documentation (https://docs.eosnetwork.com).
```

Comment:
Replace all instances of "publish" with "contribute" and simplify:
"publish new or updated content to" -> "contribute content to the"
Markdown links have the following format: \[highlighted-text](url).
Also, we don't publish to "EOS documentation" - we publish to a website or portal. However, this procedure is really about contributing to the EOS documentation in general, regardless of publishing/post-production details.
Therefore, it should be: \[EOS Documentation](https://docs.eosnetwork.com).